mantidimaging/gui/dialogs/cor_inspection/model.py

```diff
@@ -4,6 +4,8 @@
 from dataclasses import replace
 from logging import getLogger
 
+import numpy as np
```

Comment:
If we are just using `numpy` when doing type annotation then we should probably use 

```suggestion
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    import numpy as np
```
to prevent from unnecessary imports